corehq/apps/app_manager/views/modules.py

```diff
@@ -872,10 +872,13 @@ def overwrite_module_case_list(request, domain, app_id, module_unique_id):
             )
         else:
             try:
-                if detail_attrs:
+                if detail_type == "short":
```

Comment:
just to learn more, what does "short/long" really point to? I have not touched app manager for a while, was this Case list, and Case detail? Is this documented anywhere?

Reply:
Yeah, "short" is case list and "long" is case detail. I don't know that it's documented well, there are probably comments random in other places. The `DetailPair` class in app_manager.models could be a good class to document better.